examples/speaker_tasks/diarization/conf/inference/diar_infer_telephonic.yaml

```diff
@@ -29,12 +34,12 @@ diarizer:
       filter_speech_first: True 
 
   speaker_embeddings:
-    model_path: ??? # .nemo local model path or pretrained model name (titanet_large, ecapa_tdnn or speakerverification_speakernet)
+    model_path: titanet_large # .nemo local model path or pretrained model name (titanet_large, ecapa_tdnn or speakerverification_speakernet)
     parameters:
-      window_length_in_sec: 1.5 # Window length(s) in sec (floating-point number). Either a number or a list. Ex) 1.5 or [1.5,1.0,0.5]
-      shift_length_in_sec: 0.75 # Shift length(s) in sec (floating-point number). Either a number or a list. Ex) 0.75 or [0.75,0.5,0.25]
-      multiscale_weights: null # Weight for each scale. should be null (for single scale) or a list matched with window/shift scale count. Ex) [0.33,0.33,0.33]
-      save_embeddings: False # Save speaker embeddings in pickle format.
+      window_length_in_sec: [1.5,1.25,1.0,0.75,0.5] # Window length(s) in sec (floating-point number). either a number or a list. ex) 1.5 or [1.5,1.0,0.5]
+      shift_length_in_sec: [0.75,0.625,0.5,0.375,0.25] # Shift length(s) in sec (floating-point number). either a number or a list. ex) 0.75 or [0.75,0.5,0.25]
+      multiscale_weights: [1,1,1,1,1] # Weight for each scale. should be null (for single scale) or a list matched with window/shift scale count. ex) [0.33,0.33,0.33]
+      save_embeddings: True # Save speaker embeddings in pickle format.
```

Comment:
why change the default to True?

Reply:
This is needed for multiscale diarization decoder to use the embedding. We can specify it to be False for clustering diarization.

Reply:
yeah better let the user know it's fundamental for msdd, if disabled, msdd would fail.

Reply:
I added save_embedding=False example config in Readme.